tensorflow_addons/image/transform_ops.py

```diff
@@ -120,17 +123,36 @@ def transform(
                 % len(transforms.get_shape())
             )
 
-        fill_value = tf.convert_to_tensor(
-            fill_value, dtype=tf.float32, name="fill_value"
-        )
-        output = tf.raw_ops.ImageProjectiveTransformV3(
-            images=images,
-            transforms=transforms,
-            output_shape=output_shape,
-            interpolation=interpolation.upper(),
-            fill_mode=fill_mode.upper(),
-            fill_value=fill_value,
-        )
+        if LooseVersion(tf.__version__) >= LooseVersion("2.4.0"):
+            fill_value = tf.convert_to_tensor(
+                fill_value, dtype=tf.float32, name="fill_value"
+            )
+            output = tf.raw_ops.ImageProjectiveTransformV3(
+                images=images,
+                transforms=transforms,
+                output_shape=output_shape,
+                interpolation=interpolation.upper(),
+                fill_mode=fill_mode.upper(),
+                fill_value=fill_value,
+            )
+        else:
+            fill_mode = fill_mode.upper()
+            # TODO(WindQAQ): Get rid of the check once we drop TensorFlow < 2.4 support.
```

Comment:
Can we automatically/programmatically fail with https://github.com/tensorflow/addons/blob/master/setup.py#L79?
What do you think? I suppose that this kind of things will stay forever in the code :wink:

Reply:
Eh, we can clean up the codebase as our compatibility matrix moves. See https://github.com/tensorflow/addons/pull/2294

Reply:
Yes but It could be easier if we are failing in different points in the sourfe when we move the minimal version or the single version we support. Do you understand what I meant?